server/src/test/java/org/candlepin/resource/OwnerProductResourceTest.java

```diff
@@ -57,14 +58,15 @@
 public class OwnerProductResourceTest extends DatabaseTestFixture {
 
     @Inject protected ProductManager productManager;
+    @Inject protected JobManager jobManager;
```

Comment:
Something I forgot earlier, there is no need to create this field, since it is already present in the parent class `DatabaseTestFixture`.

Reply:
Done.